cis/streams.py

```diff
@@ -26,10 +26,41 @@ def publish_to_cis(data, partition_key):
 
     json_payload = json.dumps(base64_payload).encode('utf-8')
 
+    return json_payload
+
+
+def publish_to_cis(data, partition_key):
+    """
+    Publish data to CIS kinesis stream given a partition key.
+
+    :data: Data to be published to kinesis (dict)
+    :partition_key: Kinesis partition key used to publish data to
+    """
+
+    payload = prepare_payload(data)
+
     response = kinesis.put_record(
         StreamName=KINESIS_STREAM_ARN.split('/')[1],
-        Data=json_payload,
+        Data=payload,
         PartitionKey=partition_key
     )
 
     return response
+
+
+def invoke_cis_lambda(data):
```

Comment:
If the LAMBDA VALIDATOR ARN is optional can we also make this raise if the function is called and the lambda can not be found instead of relying on boto3